src/formulas/composer_tokenizer.ts

```diff
@@ -44,6 +45,7 @@ export interface EnrichedToken extends Token {
   functionContext?: FunctionContext;
   color?: Color;
   isFormulaCursorContextToken?: boolean;
+  isRelatedToCursorToken?: boolean;
```

Comment:
Same here, `isRelatedToCursorToken` doesn't mean much. We probably should have `highlight` and `lightenedColor` keys or something

Reply:
there is too much notion of different "light" on the composer, so I am not convinced by `lightenedColo`. For this also that I name by the functional meaning (but yes not necessarily better either).

To do with others maybe

Reply:
`isParenthesisLinkedToCursor` ? `isParenthesisHighlighted` ?

I'd add the notion of parenthesis in the name since it's only used for parenthesis (for now at least).
I agree that `isRelatedToCursorToken` doesn't mean much and you have to look at the implementation to know what it is/does.